packages/location-field/src/index.js

```diff
@@ -43,7 +50,25 @@ class LocationField extends Component {
     getGeocoder(geocoderConfig)
       .autocomplete({ text })
       .then(result => {
-        this.setState({ geocodedFeatures: result.features });
+        let message;
+        // If no features found in response, default to empty array.
+        let geocodedFeatures = result && result.features;
+        if (!geocodedFeatures) {
+          // Get the Pelias error message if exists.
+          // TODO: determine how other geocoders return error messages.
+          const errorMessage =
+            result &&
+            result.results &&
+            result.results.error &&
+            result.results.error.message;
+          // If the result did not contain a list of features, add special note.
+          message = "Could not reach geocoder";
+          if (errorMessage) message += ` (${errorMessage})`;
```

Comment:
Using a templated string for the entire message would have been better.

Reply:
This seems about as legible as (maybe more than) that.